client/src/app/containers/Suggestions/ChoosingCard.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { sliceKey, reducer } from 'app/containers/Suggestions/slice';
+import PropTypes from 'prop-types';
+import { useInjectReducer } from 'redux-injectors';
+
+const useStyles = makeStyles({
+  root: {
+    width: 120,
+    height: 160,
+  },
+  value: {
+    fontSize: 14,
```

Comment:
We should keep font sizes in `rem` so they stay relative to a single reference size. Root font size is probably 16, so this would become `'0.875rem'`